<!DOCTYPE html><html lang="en"><head><meta charset="utf-8"><meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no"><title>BCOM Presentation</title><link rel="stylesheet" href="css/reveal.css"><link rel="stylesheet" href="css/theme/white.css"><!-- Theme used for syntax highlighting of code--><link rel="stylesheet" href="lib/css/zenburn.css"><link rel="stylesheet" href="css/personal.css"><!-- Printing and PDF exports--><script>var link = document.createElement( 'link' );
link.rel = 'stylesheet';
link.type = 'text/css';
link.href = window.location.search.match( /print-pdf/gi ) ? 'css/print/pdf.css' : 'css/print/paper.css';
document.getElementsByTagName( 'head' )[0].appendChild( link );</script></head><body><div class="reveal"><div class="slides"><section><h3>Protein subchloroplast localization prediction: MultiP-SChlo</h3><h4>Michael Carrillo</h4></section><section><p>MultiP-SChlo: multi-Label Protein Subchloroplast Localization Prediction with Chou's Pseudo Amino Acid Composition and a Novel Multi-Label Classifier</p><p>Xiao Wang, Weiwei Zhang, Qiuwen Zhang and Guo-Zheng Li</p><p><a href="http://biomed.zzuli.edu.cn/bioinfo/multip-schlo/" target="_blank">http://biomed.zzuli.edu.cn/bioinfo/multip-schlo/</a></p></section><section><h2>Biological motivation</h2></section><section><h3>Plant cell</h3><img src="img/plant_cell.png" alt="Plant cell"><p class="imgsrc">https://upload.wikimedia.org/wikipedia/commons/thumb/d/d8/Plant_cell_structure-en.svg/1280px-Plant_cell_structure-en.svg.png</p></section><section><h3>Localization is important!</h3><p><ol><li>Several non compatible metabolic reactions happening at the same time</li><li>Proteins do not work alone. They're affected by their environment</li><li>Cell must transport proteins where they're "needed"</li></ol></p></section><section><h3>How does transportation occur?</h3><p>Through small sequences in the protein that bind to receptors that guide them</p><p>This signals are not static not unique</p></section><section><p>For investigators is necessary to know where a protein is located to understand the role of the protein in any given process</p><p>Biological experiments can be carried out to find biological evidence of these locations. However, they're expensive and not really practical</p><p>With expansion of protein sequencing new methods are needed to predict protein localization</p></section><section><p>Develop computational methods that allow investigators to <strong>predict</strong> these localizations</p></section><section><h2>Computational context</h2></section><section><h2>Statistical classification</h2><p>Machine Learning development</p><p>Identify to which a set of categories a new observation belongs based on a training set of data containing observations whose category membership is known</p></section><section><h2>Examples</h2><p><ul><li>Spam classification</li><li>Fraud detection</li><li>Protein subcellular localization prediction</li></ul></p></section><section><p><ul><li>Develop mathematical models that can predict categories for new observations</li><li>A training set that is already classified using other methods</li><li>Algorithms do not understand the nature of the problem, they find relations between inputs and output</li><li>Necessity of well distributed, abundant data </li></ul></p></section><section><h2>Linear regression</h2><div style="width: 100%"><div style="float: left; display:inline-block; width: 50%"><img src="img/linear_regression.png" alt="Linear regression"><p class="imgsrc">https://upload.wikimedia.org/wikipedia/commons/thumb/3/3a/Linear_regression.svg/400px-Linear_regression.svg.png</p></div><div style="float: right; width: 50%"><ul><li>A training set</li><li>A mathematical model</li><li>Training data is used to find parameters</li><li>Test how good is the model</li><li>Use it</li></ul></div><div style="clear: both"></div></div></section><section><h2>Models for classification</h2><ul><li><strong>Support Vector Machines</strong></li><li>Neural Networks</li><li>Decision Trees</li><li>Regression methods (Linear)</li><li>...</li></ul></section><section><h2>Support Vector Machines (SVM)</h2><img src="img/svmintuition.png" alt="SVM intuition"></section><section><h2>Characteristics of the Classifier</h2><ul><li>Non-probabilistic</li><li>Binary</li><li>Linear</li><li>Acts on points in space</li></ul></section><section><h2>Available implementation</h2><p>LIBSVM</p><p><a href="http://www.csie.ntu.edu.tw/~cjlin/libsvm/#nuandone" target="_blank">http://www.csie.ntu.edu.tw/~cjlin/libsvm/#nuandone</a></p></section><section><h2>Why this tool?</h2><ul><li>Not the first tool available</li><li>More subchloroplast localizations</li><li>Older datasets were too biased</li><li>Older tools were not multi-label</li><li>Specifically constructed for chloroplasts</li><li>UniProtKB/Swiss-Pro has 14408 chloroplast proteins, only 6955 annotated (48%)</li></ul></section><section><h2>Dataset</h2><p>UniProtKB/Swiss-Prot database (release 2013_05)</p><ol><li>Only annotated (single or multiple) were used</li><li>Ambiguous annotation was removed ('by similarity', 'probable', 'potential')</li><li>Fragments or less than 50 aminoacid proteins were removed</li><li>Proteins with ambiguous letters ('B', 'X', 'Z') were removed</li><li>Reduction of redundancy and homology bias with CD-HIT</li></ol><p>MSchlo578 was created with 578 chloroplast proteins</p></section><section><h2>Dataset</h2><img src="img/dataset.png" alt="Dataset"></section><section><h2>Labels and distribution</h2><img src="img/distribution.png" alt="Dataset"></section><section><h2>Labels and distribution</h2><img src="img/distribution2.png" alt="Dataset"></section><section><h2>Feature extraction</h2><p>Let's use the protein sequence</p><p>Sequences are not standard not really useful for ML</p></section><section><h2>Amino Acid Composition (AAC)</h2><p>A vector of 20 positions where each component represents the frequencies of each amino acid</p><p><strong>Problem?</strong></p><p>Ordering information is lost</p></section><section><h2>Pseudo Amino Acid Composition (PseAAC)</h2><p>First 20 vectors are AAC</p><p>Added &xi; * &lambda; components</p><p>In this case:</p><ul><li>&lambda; is 50</li><li>&xi; is 6</li></ul><p>Total of 20 + (50 * 6) = 320 components</p></section><section><h2>Pseudo Amino Acid Composition (PseAAC)</h2><p>&lambda; has to be less than the length of the minimum sequence. (50)</p><p>&xi; can be a subset of the following chemical properties:</p><ul><li>Hydrophobicity</li><li>Hydrophilicity</li><li>Mass</li><li>pK (alpha-COOH)</li><li>pk (NH3)</li><li>pI (at 25CÂ°)</li></ul></section><section><h2>Pseudo Amino Acid Composition (PseAAC)</h2><img src="img/pseacc.jpg" alt="PseACC representation"></section><section><h2>Proposed classifier</h2><p>SVM based multiple classifier</p><img src="img/SVMmatrix.png" alt="SVM matrix"><p>N = 578, M = 5</p></section><section><h2>How good is this classifier?</h2><p>Jackknife test</p><p>Y<sub>i</sub> is the true labels for each example, Z<sub>i</sub> is the predicted set</p><img src="img/evaluation1.png" alt="Evaluation formulas"></section><section><h2>Compared to AL-KNN</h2><img src="img/evaluation2.png" alt="Compared to AL-KNN"></section><section><h2>Compared to single label classifiers</h2><img src="img/evaluation3.png" alt="Compared to SLC"></section><section><h2>Questions?</h2></section></div></div><script src="lib/js/head.min.js"></script><script src="js/reveal.js"></script><script>Reveal.initialize({
    history: true,
    slideNumber: true,
    transition: 'none',
    dependencies: [
        { src: 'plugin/markdown/marked.js' },
        { src: 'plugin/markdown/markdown.js' },
        { src: 'plugin/notes/notes.js', async: true },
        { src: 'plugin/highlight/highlight.js', async: true, callback: function() { hljs.initHighlightingOnLoad(); } }
    ]
});</script></body></html>